examples/hybrid/TurbulenceConvectionUtils.jl

```diff
@@ -102,31 +101,81 @@ function init_tc!(Y, p, params, colidx)
 
     (; edmf, surf_params, case) = p.edmf_cache
     tc_params = CAP.turbconv_params(params)
+    thermo_params = CAP.thermodynamics_params(params)
 
     FT = eltype(edmf)
     # `nothing` goes into State because OrdinaryDiffEq.jl owns tendencies.
     state = TC.tc_column_state(Y, p, nothing, colidx)
-    thermo_params = CAP.thermodynamics_params(params)
-
     grid = TC.Grid(state)
-    FT = CC.Spaces.undertype(axes(Y.c))
-    C123 = CCG.Covariant123Vector
-    t = FT(0)
 
-    # TODO: can we simply remove this?
-    If = CCO.InterpolateC2F(bottom = CCO.Extrapolate(), top = CCO.Extrapolate())
-    @. p.edmf_cache.aux.face.ρ[colidx] = If(Y.c.ρ[colidx])
+    # parent(state.aux.face) .= NaN
+    # parent(state.aux.cent) .= NaN
 
-    # TODO: convert initialize_profiles to set prognostic state, not aux state
+    # Initialization of auxiliary variables:
+    (; ᶜp, ᶜts) = p
+    ᶜθ_liq_ice = p.edmf_cache.aux.cent.θ_liq_ice
+    ᶜq_tot = p.edmf_cache.aux.cent.q_tot # only initialized for moist profiles
+    ᶜtke = p.edmf_cache.aux.cent.tke
+    ᶜHvar = p.edmf_cache.aux.cent.Hvar # only initialized for GABLS
+    ᶜHvar .= 0
```

Comment:
You are overwriting it with 0 anyway?

Reply:
Yeah, because it is only initialized for GABLS, but the value could still be used for other cases, as long as the model requires prognostic covariances.